Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Used by docker-compose.yml to deploy the formio application
+# (When modified, you must include `--build` )
+# -----------------------------------------------------------
+
+# Use Node image, maintained by Docker:
+# hub.docker.com/r/_/node/
+FROM node:11
```

Comment:
I would prefer this to run in alpine if you think it will work.

```FROM node:lts-alpine```

Reply:
Thanks for your feedback, great idea! 

While testing it out, the dependency flow had to change a bit, notably building [bcrypt](https://www.npmjs.com/package/bcrypt) on alpine. This means this compiled version must exist in `./node_modules` which became a bit of a problem. 

It is still working, but it did increase the complexity so I added it as another commit.